fullstop/src/main/resources/schema.sql

```diff
@@ -18,4 +18,53 @@ CREATE TABLE IF NOT EXISTS fullstop_data.violation (
   CONSTRAINT unique_violation UNIQUE (account_id, region, event_id, message)
 );
 
---ALTER DEFAULT PRIVILEGES IN SCHEMA fullstop_data GRANT SELECT ON SEQUENCES TO xxx;
\ No newline at end of file
+--ALTER DEFAULT PRIVILEGES IN SCHEMA fullstop_data GRANT SELECT ON SEQUENCES TO xxx;
+CREATE TABLE IF NOT EXISTS fullstop_data.application (
+  id               BIGSERIAL NOT NULL PRIMARY KEY,
+  name             TEXT,
+  created          TIMESTAMP,
+  created_by       TEXT,
+  last_modified    TIMESTAMP,
+  last_modified_by TEXT,
+  version          BIGINT    NOT NULL,
+  CONSTRAINT unique_app_name UNIQUE (name)
+);
+
+CREATE TABLE IF NOT EXISTS fullstop_data.app_version (
+  id               BIGSERIAL NOT NULL PRIMARY KEY,
+  name             TEXT,
+  created          TIMESTAMP,
+  created_by       TEXT,
+  last_modified    TIMESTAMP,
+  last_modified_by TEXT,
+  version          BIGINT    NOT NULL,
+  CONSTRAINT unique_version_name UNIQUE (name)
+);
+
+CREATE TABLE IF NOT EXISTS fullstop_data.app_has_version (
```

Comment:
I would recommend to use a standard naming scheme here, then Hibernate/JPA could also handle this without the need to specify @JoinColumns...